x-pack/plugins/global_search_bar/public/search_syntax/parse_search_params.test.ts

```diff
@@ -44,20 +41,16 @@ describe('parseSearchParams', () => {
       filters: {
         tags: ['foo', 'dolly'],
         types: ['bar'],
-        unknowns: {},
       },
     });
   });
 
-  it('handles unknowns field clauses', () => {
```

Comment:
In the previous code, nothing was handling the `unknowns` field from the result. This PR takes the opinion that pulling out unknown field clauses so they can't be interacted with is not the user's intention.